code/modules/client/preferences.dm

```diff
@@ -1103,7 +1109,7 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 								continue
 							if(gender == FEMALE && S.gender == MALE)
 								continue
-							if( !(species in S.species_allowed))
+							if(!(species in S.species_allowed))

```

Comment:
Yeah, I think it'd group like ((!species) in S.species_allowed), which is very definitely going to smash things like crazy. The more brackets, the better, when it comes to things that will hilariously explode on a bad regrouping.